src/components/Chart/Chart.js

```diff
@@ -29,15 +34,27 @@ export const Chart = ({ data: chartInfo }) => {
     <div className="chart">
       {isVisible ? (
         <React.Fragment>
-          <div className="y-axis-label">Seasons</div>
+          <div className="y-axis-label">{chartType}</div>
           <div className="x-axis-label">Episodes</div>
           <button onClick={onToggleChart}>Toggle</button>
         </React.Fragment>
       ) : null}
       <div className="chart-data">
         {_.map(newChartInfo, (info, index) => {
           const { data, type } = info;
-          return <ChartSquare key={index} type={type} data={data} />;
+          return (
+            <ChartSquare
+              key={index}
+              type={type}
+              data={data}
+              onClick={() => {
+                if (isSidebarOpen === false) {
```

Comment:
could also re-write this if as 

```
!isSideBarOpen &&  setIsSidebarOpen(true);
```

if we want to start using simple one line if statements